Sources/Core/Log/LogOutputStream.swift

```diff
@@ -22,9 +23,11 @@
                     print("[SwiftPrettyPrint] Failed to create directory. (\(directoryPath))")
                 }
             }
+            #endif
         }
 
         func write(_ string: String) {
+#if !os(WASI)
```

Comment:
[imo]

It is preferable to align the indentation with the other lines:

```suggestion
            #if !os(WASI)
```